lib/core/elastic_connector_actions.rb

```diff
@@ -155,8 +161,8 @@ def fetch_document_ids(index_name)
           hits = response['hits']['hits']
           ids = hits.map { |h| h['_id'] }
           result += ids
-          body[:search_after] = hits.last['sort']
           break if hits.size < page_size
+          body[:search_after] = hits.last['sort']
```

Comment:
What was the bug here?

Reply:
When hits are empty, `hits.last` is `nil`

Reply:
Ahhhh, good one!